frontend/src/metabase/components/ListItem/ListItem.jsx

```diff
@@ -30,9 +31,7 @@ const ListItem = ({
         </div>
         <div className={S.itemBody}>
           <div className={S.itemTitle}>
-            <ListItemName tooltip={name} tooltipMaxWidth="100%">
```

Comment:
`tooltipMaxWidth` here was the biggest sin. I think with our old tooltip, this was based on the width of the container, and our new tooltips get appended to the body so the container is the screen width.